src/app/services/user/user-identity-provider.service.ts

```diff
@@ -1,19 +1,27 @@
 import { Injectable } from "@angular/core";
 import { IIronIdentityProvider } from '../iron/i-iron-identity-provider';
+import { AnnouncementService } from '../announcement/announcement.service';
+
+const errorMessage =
+`Oops. There doesn't appear to be a backend service to sign requests.\n
+Did you run ng serve? If so, please run npm start instead.`;
 
 @Injectable({
     providedIn: "root"
 })
 export class UserIdentityProviderService implements IIronIdentityProvider {
+    constructor(private announcementService: AnnouncementService) {
+    }
+
     /**
      * Request a signed JWT from the server for the current project and segment.
      */
     getJWT(userID): Promise<string> {
-        // TODO: Catch and gracefully handle invalid JWT
         // TODO: See if we can eliminate node server dependency
```

Comment:
Could we the comment on line 20 as well since, at this point we have decided not to eliminate the node server dependency?

Reply:
Did you mean delete the comment on line 20?

(Comment for catch and gracefully handle invalid JWT was removed because we're now doing that.)